docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,118 @@
+
+* Snapshot repositories are much more reliable than local storage on individual
+  nodes.
+
+* The monetary cost per GB stored in a snapshot repository is usually lower
+  than on a node.
+
+* The monetary cost per read or write operation on a snapshot repository is
+  usually much higher.
+
+* Reading or writing data in a snapshot repository usually takes much more time
+  compared with accessing a node's local storage.
+
+{search-snaps-cap} let you reduce your operating costs by treating the snapshot
+as the authoritative copy of some of your indices. The high reliability of the
+snapshot repository removes the need to keep multiple copies of each shard in
+your cluster purely for resiliency. {es} makes a copy of a {search-snap} onto
+the nodes in the cluster to reduce the performance impact and costs of
+accessing the snapshot repository.
+
+With {search-snaps} you may be able to halve your cluster size without
+increasing the risk of data loss or reducing the amount of data exposed to
+searches. Put differently, {search-snaps} may allow you to expose twice as much
+data to searches for a given cluster size.
+
+=== Using {search-snaps}
+
+A {search-snap} can be searched just like any other index. Since there is a
+copy of the {search-snap} on the nodes in the cluster its searches will perform
+similarly to searches on a regular index.
+
+The shards of {search-snaps} are allocated just like shards of any other index.
+You can, for instance, use <<shard-allocation-filtering>> to restrict these
+shards to a subset of your nodes.
+
+Normally you will manage {search-snaps} via the <<ilm-searchable-snapshot,
+searchable snapshots ILM action>> which automatically and transparently
+converts your index into a {search-snap} when it reaches the `cold` ILM phase.
+If you already have some snapshots that you want to search, you can also use
+the <<searchable-snapshots-api-mount-snapshot, mount snapshot API>> to manually
+mount them as {search-snaps}.
+
+You must not delete a snapshot while any of its indices are mounted as a
+{search-snap}. However, most snapshots are taken as a backup and contain a
+large number of indices, most of which will not be mounted as searchable
+snapshots. If you mount just one of these indices then you may not delete that
+snapshot. Therefore we recommend that you use the <<clone-snapshot-api, clone
+snapshot API>> to cheaply create a clone of a snapshot where the clone contains
+just the index you want to mount. This will mean you can delete the
+multiple-index snapshot, reducing the size of your snapshot repository, without
+losing access to any mounted indices.
+
+We recommend that you <<indices-forcemerge, force-merge>> indices to a single
+segment per shard before mounting them as {search-snaps}. Each read from a
+snapshot repository takes time and costs money, and the fewer segments there
+are the fewer reads are needed to restore the snapshot.
```

Comment:
I'd rather this was down here. It's not _very_ important to force-merge things before mounting them, and if you're mounting an existing snapshot you basically have no choice since you can't do anything about the segment count without restoring each index, merging it and re-snapshotting it.